docs/API.md

```diff
@@ -1,5 +1,107 @@
 # API
 
+## `createMaterialPalette()`
```

Comment:
Sorry, but that's your personal opinion about structuring and it's already well structured. The API is thorough and complete. That section doesn't have to follow `withMaterialPalette` because you did it exactly that way. We don't have any convention in place. Again, this is the first release and I am putting all my effort to make it happen. We can always sit down later and elaborate a plan for structuring docs. So please guys, I am expecting more collaboration and flexibility on your side rather than pointing out every single detail, without enough justification.